electron-component/main.js

```diff
@@ -0,0 +1,35 @@
+const WindowsManager = require("./components/WindowsManager");
+
+const electron = require('electron');
+const conf = require('./config');
+
+// Module to control application life.
+const app = electron.app;
+
+function initializeCommunicator() {
+    //the way to init the communicator instance:
+    require('./components/Communicator');
+}
+
+function createWindows() {
+    WindowsManager.addWindow(
+        conf.defaultBrowserOptions,
```

Comment:
if you're already saving the parameters in the "config" file, why not saving them all together and provide a single object to the addWindow method?

Reply:
I think its more clear to other - this separation